### PR TITLE
Add  wishlist  functionality

### PR DESCRIPTION
**Pull request by Prathyusha-96**
- Add and Remove Item from Wishlist in Products page
- Remove Item from Wishlist from Wishlist page
- Move Item to Wishlist from Cart page

### CODE REVIEW

File: src/backend/db/products.js [hunk 1]
```diff
@@ -11,7 +11,6 @@ import { longkurti,handbag,shoes,premiumjacket,kidsdress, denimjacket,kidsjeans,
     image: longkurti,
     badge: "None",
     discount: "(20% OFF)",
```
Comment (govindappaarun) [inline src/backend/db/products.js:13]: suggestion: you could have just kept number and made display string later

File: src/pages/Cart/Cart.jsx [hunk 2]
```diff
@@ -1,63 +1,153 @@
 import React from "react";
 import './cart.css'
+import { useEffect } from "react";
+import { Link } from "react-router-dom";
+import { CartItem } from "./components/CartItem";
```
Comment (govindappaarun) [inline src/pages/Cart/Cart.jsx:5]: suggestion: to reduce the too many lines for import you can add index.js file in the **components** folder similarly in **context**
with that u can do import in single line

File: src/pages/Cart/Cart.jsx [hunk 3]
```diff
@@ -1,63 +1,153 @@
...
+
+
+
+
+
+
+
+    
+    //     <div className="container" >
+
```
Comment (govindappaarun) [inline src/pages/Cart/Cart.jsx:101]: suggestion: remove commented code

**Review by govindappaarun — APPROVED** [review]
good to merge
Nice breakup of code
file and variable naming is good

**Review by BISINIGISHIVANI — APPROVED** [review]

**Review by BISINIGISHIVANI — APPROVED** [review]

**Pull request merged**